css/src/components/accordion.scss

```diff
@@ -0,0 +1,157 @@
+/* stylelint-disable selector-no-qualifying-type */
+$accordion-font-size-sm: $font-size-8 !default;
+$accordion-font-size-md: $font-size-7 !default;
+$accordion-font-size-lg: $font-size-6 !default;
+
+$accordion-spacing-xs: $spacer-2 !default;
+$accordion-spacing-sm: $spacer-3 !default;
+$accordion-spacing-md: $spacer-4 !default;
+$accordion-spacing-lg: $spacer-5 !default;
+$accordion-spacing-offset-sm: $spacer-6 !default;
+$accordion-spacing-offset-md: $spacer-7 !default;
+$accordion-spacing-offset-lg: $spacer-8 !default;
+
+$accordion-transition: transform 0.15s !default;
+
+// Spacing and font-size
+
+@mixin accordion-summary-sizes($font-size, $padding-size, $icon-offset) {
+	$bg-icon-offset: calc($icon-offset + 0.25rem);
+
+	.accordion-content,
+	summary {
+		padding-block: $padding-size;
+	}
+
+	summary {
+		font-size: $font-size;
+
+		&:not(.accordion-icon-end) {
+			padding-inline-start: $icon-offset;
+
+			&::before {
+				inset-block-start: calc($control-padding-horizontal + $padding-size);
+			}
+		}
+
+		&.accordion-icon-end {
+			padding-inline-end: $icon-offset;
+
+			&::after {
+				inset-block-start: calc($padding-size * 1.5);
+			}
+		}
+	}
+
+	.accordion-content {
+		padding-block: $padding-size;
+
+		&[class*='background-color-'] {
+			padding-inline: $padding-size;
+		}
+	}
+
+	&[class*='background-color-'] {
+		summary,
+		.accordion-content {
+			padding-inline: $padding-size;
+		}
+
+		summary {
+			&:not(.accordion-icon-end) {
+				padding-inline-start: $bg-icon-offset;
+
+				&::before {
+					inset-inline-start: $padding-size;
+				}
+			}
+
+			&.accordion-icon-end {
+				padding-inline-end: $bg-icon-offset;
+
+				&::after {
+					inset-inline-end: calc($accordion-spacing-xs + $padding-size);
+				}
+			}
+		}
+	}
+}
+
+.accordion {
+	@include accordion-summary-sizes(
+		$accordion-font-size-md,
+		$accordion-spacing-md,
+		$accordion-spacing-offset-md
+	);
+
+	line-height: 1.5;
```

Comment:
This will override the line-height inside of the `.accordion-content`, instead of using our typography defaults. If this is meant for the summary, could we scope it to that directly?

Reply:
Good call, updated ✅